package/index.ts

```diff
@@ -6540,4 +6540,13 @@ export function createElement(name: any, props: any): IBobrilNode {
     }
 }
 
+export interface IFragmentData {
+    children: IBobrilChildren,
+    key?: string
+}
+
+export function Fragment(data?: IFragmentData){
+    return { ...data };
```

Comment:
Why do you think it needs cloning? why not simple return data?

Reply:
No special need. I was thinking about such usage as well. Cloning was added just because of keeping separated concepts of data and nodes.